tox.ini

```
@@ -0,0 +1,12 @@
+# For more information about tox, see https://tox.readthedocs.io/en/latest/
+[tox]
+envlist = py27,py34,py35,py36,py37,pypy,flake8
```

`setup.py` says only Python 3 is supported.

```suggestion
envlist =py34,py35,py36,py37,pypy3,flake8
```